### PR TITLE
Nightingale conversion fixes and additional updates

### PR DESCRIPTION
**Pull request by pkmitre**
This pull request makes a handful of not-necessarily-related changes:

1. Updates and adds to the code for converting from the Nightingale data format to FHIR
2. Adds the CertificationRoleHelper method
3. Documents the new helper methods in the README
4. Copies the Connectathon record generation code into VRDR from Canary so that it can be accessed from the VRDR.CLI; once this is merged the version of the file in Canary can be removed and Canary can refer to this version
5. Adds functionality to the VRDR.CLI application for generating Connectathon records from the command line
6. Adds a ruby script for generating a large quantity of FHIR records using the Connectathon records
7. Adds a ruby script for converting an Excel file with records in pseudo-IJE format into a set of FHIR records for testing
8. Removes support for the now-obsolete netcoreapp2.1

### CODE REVIEW

File: VRDR.HTTP/Nightingale.cs [hunk 1]
```diff
@@ -610,54 +647,78 @@ private static void SetStringValueDeathRecordDictionary(DeathRecord deathRecord,
             prop.SetValue(deathRecord, dict, null);
         }
 
-        /// <summary>Set a Yes/No (coded) value on a DeathRecord.</summary>
-        private static void SetYesNoValueDeathRecordCode(DeathRecord deathRecord, string property, string value)
+        /// <summary>Set a Yes/No (coded) value on a DeathRecord using Boolean helpers.</summary>
+        private static void SetYesNoValueDeathRecordBoolean(DeathRecord deathRecord, string property, string value)
```
Comment (smacadam) [inline VRDR.HTTP/Nightingale.cs:651]: It seems like this function is useful for other parts of the library, should it be in VRDR vs VRDR.HTTP?

File: VRDR.HTTP/Nightingale.cs [hunk 2]
```diff
@@ -610,54 +647,78 @@ private static void SetStringValueDeathRecordDictionary(DeathRecord deathRecord,
...
+            }
+            else
+            {
+                prop.SetValue(deathRecord, null);
             }
         }
 
         /// <summary>Set a Yes/No (coded) value on a DeathRecord.</summary>
-        private static void SetYesNoValueDeathRecordBool(DeathRecord deathRecord, string property, string value)
+        private static void SetYesNoValueDeathRecordCode(DeathRecord deathRecord, string property, string value)
```
Comment (smacadam) [inline VRDR.HTTP/Nightingale.cs:674]: Same as question above

**Review by smacadam — COMMENTED** [review]
Questions on where the Nightingale helper functions should live within the library

File: VRDR.HTTP/Nightingale.cs (hunk 1 above)
Comment (pkmitre) [reply VRDR.HTTP/Nightingale.cs:651]: Seems reasonable, but should only be moved when it's actually first used outside of VRDR.HTTP (YAGNI principle)

**Review by smacadam — APPROVED** [review]

**Pull request merged**